packages/rest/src/providers/reject.provider.ts

```diff
@@ -6,15 +6,19 @@
 import {LogError, Reject, HandlerContext} from '../types';
 import {inject, Provider} from '@loopback/context';
 import {HttpError} from 'http-errors';
-import {writeErrorToResponse} from '../writer';
 import {RestBindings} from '../keys';
+import * as errorHandler from 'strong-error-handler';
 
-const debug = require('debug')('loopback:rest:reject');
+export const defaultErrorHandlerOptions: errorHandler.ErrorHandlerOptions = {
+  log: false,
+};
 
 export class RejectProvider implements Provider<Reject> {
   constructor(
     @inject(RestBindings.SequenceActions.LOG_ERROR)
     protected logError: LogError,
+    @inject(RestBindings.ERROR_HANDLER_OPTIONS)
+    protected errorHandlerOptions?: errorHandler.ErrorHandlerOptions,
```

Comment:
If the options are optional (as you specified in the type annotation), then you also need to configure `@inject` to be optional. In which case it's better to use property-level injection instead of constructor-level injection.

Alternatively, keep the options required and fix the typescript signature (remove the question mark).